.changeset/weak-sheep-dance.md

```diff
@@ -0,0 +1,4 @@
+---
+"@vercel/build-utils": patch
+"@vercel/client": patch
+---
```

Comment:
Not sure what we usually do about this, but this change isn't ready to be used yet so I don't want it to show up in the changelogs

Reply:
Both would be minor IMO. build-utils introduces a new exported function. client supports a new value for the `archive` option.